tms/src/main/java/com/netcracker/edu/tms/web/Controller.java

```diff
@@ -0,0 +1,90 @@
+package com.netcracker.edu.tms.web;
+
+import com.netcracker.edu.tms.model.Priority;
+import com.netcracker.edu.tms.model.Status;
+import com.netcracker.edu.tms.model.Task;
+import com.netcracker.edu.tms.service.TaskService;
+import org.apache.velocity.exception.ResourceNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.math.BigInteger;
+import java.text.ParseException;
+import java.util.List;
+
+@RequestMapping(value = "/tasks")
+@RestController
+public class Controller {
+    @Autowired
+    private TaskService taskService;
+
+    @GetMapping("/getTaskById")
```

Comment:
@GetMapping("/{id}")
public ResponseEntity<Task> getTaskById(@PathVariable BigInteger id)

minimalistic style is preferred like we did in tasks before